### PR TITLE
Add TOC of talks 

### LINKED ISSUE #81: TalksのIndexページに目次が欲しい
## 解決したいこと

image

Talksページにアクセスすると、初日の最初のトークが表示されます。2日目になると初日のトークより2日目のトークへのアクセスがしやすいと嬉しい気持ちです。手動でスクロールしなくても2日目のトークをシュッと表示できるように、目次を追加することを提案します。

### PR DESCRIPTION
**Pull request by megane42**
- fixes #81 
- I'm not picky about the appearance, so please feel free to tweak some of the UI :)
- I am concerned that I cannot return to the TOC by pressing the browser back button after clicking anchor links...

### closed

image

### opened

image

### closed

image

### opened

image

### CODE REVIEW

File: app/helpers/talks_helper.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module TalksHelper
+  def time_with_zone_to_anchor(time_with_zone)
+    time_with_zone.in_time_zone("Tokyo").strftime("%m%d-%H%M")
+  end
+
+  def time_with_zone_to_header(time_with_zone)
+    time_with_zone.in_time_zone("Tokyo").strftime("%d日 %H:%M ～")
```
Comment (unasuke) [inline app/helpers/talks_helper.rb:7]: @megane42 I think "27日 XX:XX～" in "27日" details is be bit verbose. What do you think about removing `%d日` from here?

Comment (megane42) [reply app/helpers/talks_helper.rb]: My intention was to match the link text with the heading text, but you have a point. I made the change so how do you think it looks?

[5b8310d](https://github.com/kaigionrails/conference-app/pull/83/commits/5b8310d2c8b6d4544dd3af3842f97458f7a9a3f7)

Comment (unasuke) [reply app/helpers/talks_helper.rb]: 👍 👍 👍 👍 👍 👍

Comment (megane42) [reply app/helpers/talks_helper.rb]: Additionally, I don't know why I use `to_s` at `<summary>`, so I fixed and made `<summary>` text plus `<a>` text is equal to a slot header text now.

58ce2232e

By the way, if I should squash commits I'll do it.

**Review by unasuke — APPROVED** [review]
TOC is beneficial! Thank you!

**Pull request merged**